cmd/ttn-lw-cli/commands/contact_info.go

```diff
@@ -109,6 +109,8 @@ var contactInfoFlags = util.FieldFlags(&ttnpb.ContactInfo{})
 var (
 	errContactInfoExists           = errors.DefineAlreadyExists("contact_info_exists", "contact info already exists")
 	errMatchingContactInfoNotFound = errors.DefineAlreadyExists("contact_info_not_found", "matching contact info not found")
+	errNoReference                 = errors.DefineInvalidArgument("missing_ref", "missing reference")
```

Comment:
- "no" vs "missing"
- "ref" vs "reference"

The reason why we need full consistency is to trace errors back to where they're returned.